config/run-lint.sh

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I suggest we leave the pylint check as it is and merge the script with my PR. So that there will not be any conflicts.

Reply:
done